### DISCUSSION TITLE
Help setting up

### DISCUSSION THREAD

**Discussion by GavinLau15** [leonhma/obsidian-functionplot#35] [Q&A] [answered]
Hi,

I recently installed this plugin you made, but when I press ctrl + p to enter my command pallete and try to type the command to create a graph, the option simply does not pop up, any help would be appreciated.

**Comment by leonhma** [comment]
Hello. Can you go into 'settings > community plugins' and check that 'Obsidian Functionplot' is enabled (the toggle is on). If that doesn't help, please check which version of the plugin you are on. The modal is only available after 1.1.0.

**Reply by GavinLau15** [reply]
Hi currently I am on version 1.2.1, and the plugin should be enabled. I attached a screenshot of what I am seeing.
![image](https://user-images.githubusercontent.com/107590444/189776315-073a96d9-6dc0-4dc5-b02b-47b3dc6efc33.png)

**Comment by leonhma** [comment]
Okay @GavinLau15 I am moving this to a new issue since this seems to be a bug. -> #36

**Comment by leonhma** [comment, marked as answer]
Remember to be in editor mode while trying to open the modal!
